deepchecks/nlp/text_data.py

```diff
@@ -292,7 +292,7 @@ def label(self) -> TTextLabel:
         return self._label
 
     @property
-    def classes(self) -> t.Optional[t.List[t.Union[str, int]]]:
+    def classes(self) -> t.Optional[t.List[t.Union[str, int]]]:  # Bressler - should keep or use context.model_classes?
```

Comment:
Seems like if there is a model_classes argument in run() then there is no need for a classes argument for TextData. This means that for multilabel, model_classes should receive the names the classes should have (because again in multilabel we have a binary matrix so we don't know what the labels are)
Following that logic the logic here for token classification should go into the infer_observed_labels method, no?